packages/server/src/sdk/app/rows/search/internal/sqs.ts

```diff
@@ -294,7 +295,7 @@ export async function search(
   table: Table,
   opts?: { retrying?: boolean }
 ): Promise<SearchResponse<Row>> {
-  let { paginate, query, ...params } = options
+  let { paginate, query, ...params } = cloneDeep(options)
```

Comment:
This `cloneDeep` is to avoid mutations to `options` percolating up and out of this function call.